core/.changelog.d/2755.added

```diff
@@ -0,0 +1 @@
+Deal with non-breaking spaces.
```

Comment:
Not really ideal changelog entry 😄

Reply:
> Not really ideal changelog entry 😄

Should I just remove the changelog? Does a changelog make sense for this feature?

Reply:
Removed it completely...